.tx/config

```diff
@@ -1,14 +1,15 @@
 [main]
 host = https://www.transifex.com
 
-[edx-platform.commerce-coordinator]
+[o:open-edx:p:edx-platform:r:commerce-coordinator]
 file_filter = commerce-coordinator/conf/locale/<lang>/LC_MESSAGES/django.po
 source_file = commerce-coordinator/conf/locale/en/LC_MESSAGES/django.po
 source_lang = en
-type = PO
+type        = PO
```

Comment:
is this required space for indent? @UsamaSadiq

Reply:
The `tx migrate` command executed with the new `transifex client v3` automatically makes these changes.